### PR TITLE
[#5294] Add basic topic commands to Gravitino CLI

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

 Add basic topic commands to list, create and delete topics.

### Why are the changes needed?

To extend the Gravitino CLI.

Fix: #5294

### Does this PR introduce _any_ user-facing change?

No, but it extends the Gravitiuno CLI.

### How was this patch tested?

Tested locally with a local instance of Apache Kafka.

### CODE REVIEW

**Comment by justinmclean** [comment]
@shaofengshi once the owner code is merger this is ready for review or you can review it ignoring the owner code

File: docs/cli.md
Comment (shaofengshi) [inline docs/cli.md:514]: 1. please remove the "--metalake", be consistent with others;
2. shall the "--name" be the full entity name, like "kafka.default.topic3"?

Comment (justinmclean) [reply docs/cli.md:514]: The owner code has not been merged yet and that will be fixed when that happens.

We could use the full entity name and drop the --topic flag, but it may confuse users as catalogs and topics are at the same level. I am okay with either way to do it.

Comment (justinmclean) [reply docs/cli.md]: As mentioned above @shaofengshi I am fine with it being either way, which do you prefer?

File: docs/cli.md
Comment (shaofengshi) [inline docs/cli.md:605]: this should be in the same style as table's:
"gcli table details --name catalog_postgres.hr.departments"

Why not merge the "topic" name into "name" option?

File: clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java
Comment (shaofengshi) [inline clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java:68]: "display audit information", "Display index information"; Some of the comment are "display", some are "Display", can you make them consistent?

File: docs/cli.md
Comment (shaofengshi) [inline docs/cli.md:33]: Here also, "display" and "Display".

File: docs/cli.md
Comment (justinmclean) [reply docs/cli.md:605]: Yes, that is what I asked you about above.

File: clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java
Comment (justinmclean) [reply clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java:68]: can do - although I think that was fixed in another PR/may be a merge issue

File: docs/cli.md
Comment (justinmclean) [reply docs/cli.md:623]: See my unanswered question above. It can be done either way, and I don't have a strong preference, although it might be confusing to the user to equate topics and tables.

File: clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java
Comment (shaofengshi) [reply clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java:68]: can you fix it now? I don't think it is a huge work.

File: docs/cli.md
Comment (shaofengshi) [reply docs/cli.md:605]: My comment is to keep consistent with table commands. As the entity name is different, users won't confuse.

File: clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java
Comment (justinmclean) [reply clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java:67]: Sure and done.

**Comment by justinmclean** [comment]
@shaofengshi all changes done.

**Comment by shaofengshi** [comment]
@justinmclean Justin, after merge the fileset PR, this one has conflict again, could you please merge it again? thanks!

**Comment by justinmclean** [comment]
@shaofengshi done

**Review by shaofengshi — APPROVED** [review]
LGTM

**Pull request merged**
